external/webrtc-sdk.js

```diff
@@ -29473,20 +29482,14 @@ define('WebRTC_SDK/manager/RTCLocalMediaManager',['require','my.Class','undersco
                 deferred.resolve();
                 return deferred.promise;
             }
-            if(!_.isNull(stream) && !_.isUndefined(stream)) {
-
-                Logger.debug('RTCLocalMediaManager: ' + (mute ? 'Muting ' : 'Unmuting ') + 'local audio stream now');
-
-                stream.getAudioTracks().forEach(function (track) {
-		            track.enabled = !mute;
-                });
+            Logger.debug('RTCLocalMediaManager: ' + (mute ? 'Muting ' : 'Unmuting ') + 'local audio stream now');
 
-                this.model.set('localAudioMuted', mute);
-                Logger.debug('RTCLocalMediaManager: Local Audio ', (mute ? 'muted' : 'unmuted'));
+            stream.getAudioTracks().forEach(function (track) {
+            track.enabled = !mute;
+            });
 
-            } else {
-                Logger.warn('Local Audio stream is null, cannot mute/unmute local audio');
-            }
+            this.model.set('localAudioMuted', mute);
+            Logger.debug('RTCLocalMediaManager: Local Audio ', (mute ? 'muted' : 'unmuted'));
```

Comment:
The diff looks weird here because we've removed a level of indentation, but this is exactly the same as [this](https://github.com/mirainc/sdk-webrtc-meetings/pull/5/files#diff-7c9d62cf9e59419121e70349e9a94586L29478-L29485)